recipes/jax-finufft/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_SDK_VERSION:        # [osx and x86_64]
```

Comment:
```suggestion
c_stdlib_version:        # [osx and x86_64]
```

Reply:
If I understand the linked doc properly, I think this should replace `MACOSX_DEPLOYMENT_TARGET` instead of `MACOSX_SDK_VERSION`, right? I must admit that I'm not totally sure I know when one or the other needs to be set. I know that nanobind requires `MACOSX_DEPLOYMENT_TARGET>=10.14` as [mentioned in the nanobind docs](https://nanobind.readthedocs.io/en/latest/packaging.html#step-6-build-wheels-in-the-cloud). Should I remove the `SDK_VERSION` constraint?

Reply:
Yes, sorry, this should only be applied to the deployment target.